packages/tables/src/Concerns/CanSelectColumns.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Filament\Tables\Concerns;
+
+use Filament\Forms;
+use Filament\Forms\ComponentContainer;
+use Filament\Tables\Columns\Column;
+use Filament\Tables\Filters\Filter;
+use Illuminate\Database\Eloquent\Builder;
+
+/**
+ * @property ComponentContainer $tableColumnSelectionForm
+ */
+trait CanSelectColumns
+{
+    public $tableColumnSelections = [];
+
+    public function isTableColumnSelectable(): bool
+    {
+        return count($this->getCachedSelectableColumns());
+    }
+
+    public function getTableColumnSelectionForm(): Forms\ComponentContainer
```

Comment:
`ComponentContainer` is already in the namespace so can use that instead of relative